src/main/scala/com/codacy/client/bitbucket/v2/service/WorkspaceServices.scala

```diff
@@ -1,18 +1,20 @@
 package com.codacy.client.bitbucket.v2.service
 
 import java.net.URLEncoder
-
+import com.codacy.client.bitbucket.util.UrlHelper._
 import com.codacy.client.bitbucket.client.{BitbucketClient, PageRequest, Request, RequestResponse}
 import com.codacy.client.bitbucket.v2.{Workspace, WorkspacePermission}
 
 class WorkspaceServices(client: BitbucketClient) {
 
-  def list(pageRequest: Option[PageRequest] = None): RequestResponse[Seq[Workspace]] = {
+  def list(pageRequest: Option[PageRequest] = None, pageLength: Option[Int]): RequestResponse[Seq[Workspace]] = {
```

Comment:
Why don't we have a default value here?

Reply:
No special reason, what do you think the default should be here?

Reply:
I don't know. I don't have full context, but found it strange that the other method you changed has the same signature but different defaults.

Reply:
Actually you're right, I should have the None default here aswell, otherwise this will be a breaking change, I thought you were meaning putting an actual value here